run.sh

```diff
@@ -1,26 +1,145 @@
-#!/usr/bin/env bash
+#!/bin/bash
+pokebotpath="/root/PokemonGo-Bot"
+backuppath="/root/PokemonGo-Bot/backup"
 
-# Starts PokemonGo-Bot
-config=""
+function Pokebotupdate () {
+cd $pokebotpath
+git pull
+git submodule init
+git submodule foreach git pull origin master
+virtualenv .
+source bin/activate
+pip install -r requirements.txt
+}
 
-if [ ! -z $1 ]; then
-    config=$1
+function Pokebotencrypt () {
+echo "Start to make encrypt.so"
+wget http://pgoapi.com/pgoencrypt.tar.gz
+tar -xf pgoencrypt.tar.gz 
+cd pgoencrypt/src/ 
+make
+mv libencrypt.so $pokebotpath/encrypt.so
+cd ../..
+rm -rf pgoencrypt.tar.gz
+rm -rf pgoencrypt
+}
+
+function Pokebotconfig () {
+cd $pokebotpath
+read -p "1.google 2.ptc 
+" auth
+read -p "Input username 
+" username
+read -p "Input password 
+" -s password
+read -p "
+Input location 
+" location
+read -p "Input gmapkey 
+" gmapkey
+cp configs/config.json.example configs/config.json
+if [ "$auth" = "2" ]
+then
+sed -i "s/google/ptc/g" configs/config.json
+fi
+sed -i "s/YOUR_USERNAME/$username/g" configs/config.json
+sed -i "s/YOUR_PASSWORD/$password/g" configs/config.json
+sed -i "s/SOME_LOCATION/$location/g" configs/config.json
+sed -i "s/GOOGLE_MAPS_API_KEY/$gmapkey/g" configs/config.json
+echo "edit configs/config.json to modify any other config."
+}
+
+function Pokebotinstall () {
+cd $pokebotpath
+echo "1.Debian/Ubuntu"
+echo "2.Centos/RedHat"
+echo "3.Mac os"
+echo "4.Other"
```

Comment:
I can't fix it because I don't have mac or centos devices. It's just working. Can you help me to fix it? Thx.